cmdline/tool/src/org/netbeans/modules/jackpot30/cmdline/Main.java

```diff
@@ -130,7 +130,7 @@ public class Main {
     private static final String OPTION_FAIL_ON_WARNINGS = "fail-on-warnings";
     private static final String RUN_TESTS = "run-tests";
     private static final String SOURCE_LEVEL_DEFAULT = "1.7";
```

Comment:
nitpick: `8` would be a better default.
`--help` mentions the default too, but it likely uses this field for the help text - haven't checked.